docs/UserGuide.md

```diff
@@ -141,26 +153,27 @@ Format: `create n/NAME p/PHONE_NUMBER [e/EMAIL] [a/ADDRESS] [ds/DESCRIPTION] [nw
 
 ![create](images/UserGuide/create.png)
 
-:bulb: **Note:**
-`NAME` and `PHONE_NUMBER` are the only compulsory inputs. Other parameters can be left blank.
+<div markdown="block" class="alert alert-info">
 
 :bulb: **Note:**
-It is recommended to only have 1`TAG` per client.
+- `NAME` and `PHONE_NUMBER` are the only compulsory inputs. Other parameters can be left blank.
+- It is recommended to only have 1`TAG` per client.
                   
-
-* If you have multiple meeting times with your client, simply repeat the field `mt/TIME`.
-* Does not allow you to create a person with the same name and phone number as a current person in the FABook.
-* Allows you to create a person with same name but different phone number or same number and different name as a current person in the FABook.
+</div>
```

Comment:
I think we should use a different colour to separate things like Notes and Command Shortcuts. 
I made the changes according to this in my PR but open to discussion